examples/hoa-renderer.html

```diff
@@ -0,0 +1,171 @@
+<!--
+  Copyright 2017 Google Inc. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Example: HOARotator</title>
+  <script src="audiocontext-shim.js"></script>
+  <script src="../build/omnitone.js"></script>
+  <link rel="stylesheet" href="example.css">
+  <style>
+    .slider-width500
+    {
+      width: 500px !important;
+    }
+  </style>
+</head>
+<body>
+  <h1>Example: HOARenderer</h1>
+
+  <div id="secSource">
+    <p>NOTE: Use headphones for the full-sphere surround sound.</p>
+    <h2>Select the sound source to play.</h2>
+    Azimuth: <span id="aziLabel">0</span><br/>
+    <input class="slider-width500" oninput="rotateScene();" onchange="rotateScene();" id="azi" type=range min="-180" max="180" value="0" step="1" /><br />
+    Elevation: <span id="eleLabel">0</span><br/>
+    <input class="slider-width500" oninput="rotateScene();" onchange="rotateScene();" id="ele" type=range min="-90" max="90" value="0" step="1" /><br />
+    Gain (dB): <span id="gainLabel">0</span><br/>
+    <input class="slider-width500" oninput="adjustGain();" onchange="adjustGain();" id="gain" type="range" min="-60" max="20" value="0" step="1" /><br />
+    <button id="eButtonPlayback" disabled>Play</button>
+  </div>
+
+  <script>
+    var hoaRenderer;
+    var audioSources = [
+      {name : 'pt1', url : 'resources/single_source_toa_pt1.wav'},
+      {name : 'pt2', url : 'resources/single_source_toa_pt2.wav'},
+      {name : 'pt3', url : 'resources/single_source_toa_pt3.wav'},
+      {name : 'pt4', url : 'resources/single_source_toa_pt4.wav'}];
+    var audioBufferPlayback = new Array(4);
+    var audioBufferPlaybackStartTime = 0;
```

Comment:
Also please consider move some variables to the local scope. Less global is always better. :)